.claude/skills/git-fix-pr-reviews/SKILL.md

```
@@ -28,12 +28,17 @@ User input: $ARGUMENTS
   > No PR found. Run this command on a branch with an associated PR, or specify a PR number.
 
-### Step 2: Fetch review comments
+### Step 2: Fetch review comments and thread mapping
 
-Get repository info with `gh repo view --json owner,name --jq '.owner.login + "/" + .name'` and call the following two APIs **in parallel**:
+Get repository info with `gh repo view --json owner,name --jq '.owner.login + "/" + .name'` and call the following three APIs **in parallel**:
 
 1. `gh api --paginate repos/{owner}/{repo}/pulls/{number}/comments` — inline comments (each has an `id` field needed for replies)
 2. `gh api --paginate repos/{owner}/{repo}/pulls/{number}/reviews` — review summaries (general comments)
+3. Thread-to-comment mapping via GraphQL — fetch all review threads with their comment node IDs:
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)